apps/src/code-studio/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -49,17 +51,17 @@ var WorkshopAttendance = React.createClass({
       method: "GET",
       url: `/api/v1/pd/workshops/${this.props.params.workshopId}/attendance`,
       dataType: "json"
-    }).done(function (data) {
+    }).done((data) => {
```

Comment:
note: for cases where you have a single param like this, the parens are unnecessary (but fine), i.e. it could be
```
}).done(data => {
```